### DISCUSSION TITLE
posts/cryptograpy/chacha20/

### DISCUSSION THREAD

**Discussion by giscus** [michaelneuper/website#41] [General]
# posts/cryptograpy/chacha20/

ChaCha20 THUẬT TOÁN MÃ DÒNG CHACHA20 Thuật toán mã dòng ChaCha20, quá trình mã hóa của nó được mô tả như trong Hình 1. Đầu vào của ChaCha20 bao gồm các thông số sau: 128 bit hằng số, 256 bit khóa, 32 bit bộ đếm khởi tạo có thể thiết lập bằng bất cứ giá trị nào nhưng thường là 0 hoặc 1, 96 bit nonce, bản rõ có chiều dài tùy ý.

https://vanluongkma.github.io/posts/cryptograpy/chacha20/

**Comment by trananhnhatviet** [comment]
Em fan anhhh
